deploy/cr.yaml

```diff
@@ -309,6 +310,21 @@ spec:
 #     - 10.0.0.0/8
 #   serviceAnnotations:
 #     service.beta.kubernetes.io/aws-load-balancer-backend-protocol: http
+  logcollector:
+    enabled: false
```

Comment:
should be enabled by default according to spec

Reply:
I suggest to enable it as soon as we have log rotation

Reply:
it was done